tests/integrations/coprocessor/test_select.rs

```diff
@@ -107,7 +107,7 @@ fn test_stream_batch_row_limit() {
     let req = DAGSelect::from(&product).build();
     assert_eq!(req.get_ranges().len(), 1);
 
-    let mut expected_ranges_last_byte = vec![(0, 3), (3, 6), (6, 255)];
+    let mut expected_ranges_last_byte = vec![(0, 3), (3, 6), (6, 9)];
```

Comment:
This change may be not necessary any more. However my local environment is broken now.... will be updated later if necessary.